test/tools/jsontests/TransactionTests.cpp

```diff
@@ -159,115 +158,76 @@ json_spirit::mValue doTransactionTests(json_spirit::mValue const& _input, bool _
-{
-	std::string fillersPath = dev::test::getTestPath() + "/src/TransactionTestsFiller";
-	if (!dev::test::Options::get().filltests)
-		dev::test::executeTests("ttWrongRLPTransaction", "/TransactionTests", "/TransactionTestsFiller", dev::test::doTransactionTests);
-	else
-	{
-		dev::test::TestOutputHelper::initTest();
-		dev::test::copyFile(fillersPath + "/ttWrongRLPTransaction.json", dev::test::getTestPath() + "/TransactionTests/ttWrongRLPTransaction.json");
+		if ((casename == "ttWrongRLPFrontier" || casename == "ttWrongRLPHomestead") && test::Options::get().filltests)
+			copyAllFilesFromFolder(casename);
+		else
+			fillAllFilesInFolder(casename);
 	}
-}
 
-BOOST_AUTO_TEST_CASE(tt10mbDataFieldHomestead)
-{
-	if (test::Options::get().all)
+	void fillAllFilesInFolder(string const& _folder)
 	{
-		auto start = chrono::steady_clock::now();
+		std::string fillersPath = test::getTestPath() + "/src/TransactionTestsFiller/" + _folder;
 
-		dev::test::executeTests("tt10mbDataField", "/TransactionTests/Homestead", "/TransactionTestsFiller/Homestead", dev::test::doTransactionTests);
+		string filter = test::Options::get().singleTestName.empty() ? string() : test::Options::get().singleTestName + "Filler";
+		std::vector<boost::filesystem::path> files = test::getJsonFiles(fillersPath, filter);
+		int fileCount = files.size();
 
-		auto end = chrono::steady_clock::now();
-		auto duration(chrono::duration_cast<chrono::milliseconds>(end - start));
-		cnote << "test duration: " << duration.count() << " milliseconds.\n";
+		if (test::Options::get().filltests)
+			fileCount *= 2; //tests are checked when filled and after they been filled
+		test::TestOutputHelper::initTest(fileCount);
+
+		for (auto const& file: files)
+		{
+			test::TestOutputHelper::setCurrentTestFileName(file.filename().string());
+			test::executeTests(file.filename().string(), "/TransactionTests/"+_folder, "/TransactionTestsFiller/"+_folder, dev::test::doTransactionTests);
+		}
+
+		test::TestOutputHelper::finishTest();
 	}
-	else
-		cnote << "tt10mbDataFieldHomestead skipped because --all option was not specified.\n";
-}
 
-BOOST_AUTO_TEST_CASE(tt10mbDataField)
-{
-	if (test::Options::get().all)
+	void copyAllFilesFromFolder(string const& _folder)
 	{
-		auto start = chrono::steady_clock::now();
-
-		dev::test::executeTests("tt10mbDataField", "/TransactionTests", "/TransactionTestsFiller", dev::test::doTransactionTests);
+		std::string fillersPath =  dev::test::getTestPath() + "/src/TransactionTestsFiller/" + _folder;
+		std::vector<boost::filesystem::path> files = test::getJsonFiles(fillersPath);
 
-		auto end = chrono::steady_clock::now();
-		auto duration(chrono::duration_cast<chrono::milliseconds>(end - start));
-		cnote << "test duration: " << duration.count() << " milliseconds.\n";
+		for (auto const& file : files)
+		{
+			dev::test::TestOutputHelper::initTest();
+			string copyto = dev::test::getTestPath() + "/TransactionTests/" + _folder + "/" + file.filename().string();
```

Comment:
Try to use `std::filesystem::path`.  Also, avoid using `+ "/" +` to concatenate paths.  There is an operator `/` defined between paths.